### PR TITLE
systemtests monitoring

### PR DESCRIPTION
**Pull request by famartinrh**

### CODE REVIEW

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.MonitoringTest

File: systemtests/src/main/java/io/enmasse/systemtest/cmdclients/KubeCMDClient.java [hunk 1]
```diff
@@ -290,9 +291,24 @@ public static ExecutionResultData applyFromFile(String namespace, Path path) {
...
+        return execute(Arrays.asList(CMD, "-n", namespace, "delete", "-f", path.toString()), DEFAULT_SYNC_TIMEOUT, true);
+    }
+
     public static String getMessagingEndpoint(String namespace, String addressspace) {
         Objects.requireNonNull(namespace);
         Objects.requireNonNull(addressspace);
         return execute(Arrays.asList(CMD, "-n", namespace, "get", "addressspace", addressspace, "-o", "jsonpath={.status.endpointStatuses[?(@.name==\"messaging\")].externalHost}"), DEFAULT_SYNC_TIMEOUT, true, false).getStdOut();
     }
+
+    public static ExecutionResultData labelResource(String resource, String resourceName, String key, String value) {
```
Comment (kornys) [inline systemtests/src/main/java/io/enmasse/systemtest/cmdclients/KubeCMDClient.java]: why do you label resources using executor, it isn't possible via fabric8 client?

File: systemtests/src/main/java/io/enmasse/systemtest/cmdclients/KubeCMDClient.java [hunk 2]
```diff
@@ -290,9 +291,24 @@ public static ExecutionResultData applyFromFile(String namespace, Path path) {
...
+        return execute(Arrays.asList(CMD, "label", resource, resourceName, String.format("%s=%s", key, value)), DEFAULT_SYNC_TIMEOUT, true);
+    }
+
+    public static ExecutionResultData deleteNamespace(String name) {
```
Comment (kornys) [inline systemtests/src/main/java/io/enmasse/systemtest/cmdclients/KubeCMDClient.java]: why do you remove namespace using executor, it isn't possible via fabric8 client?

File: systemtests/src/main/java/io/enmasse/systemtest/cmdclients/KubeCMDClient.java (hunk 1 above)
Comment (famartinrh) [reply systemtests/src/main/java/io/enmasse/systemtest/cmdclients/KubeCMDClient.java]: changed

**Comment by famartinrh** [comment]
@enmasse-ci run tests testcase=**.MonitoringTest

File: systemtests/src/main/java/io/enmasse/systemtest/cmdclients/KubeCMDClient.java (hunk 2 above)
Comment (famartinrh) [reply systemtests/src/main/java/io/enmasse/systemtest/cmdclients/KubeCMDClient.java]: to delete a namespace is necessary to wait for the deletion, fabric8 client doesn't wait until namespace is deleted and making it wait would require a wait loop, so I think the code is good as is using KubeCMDClient

Comment (kornys) [reply systemtests/src/main/java/io/enmasse/systemtest/cmdclients/KubeCMDClient.java]: makes sense :)

**Pull request merged**